ironfish/src/rpc/routes/wallet/burnAsset.ts

```diff
@@ -60,10 +62,15 @@ router.register<typeof BurnAssetRequestSchema, BurnAssetResponse>(
       throw new ValidationError('Invalid burn amount')
     }
 
+    const assetBuffer = Buffer.from(request.data.assetId, 'hex')
```

Comment:
this is not an asset buffer, it's the asset id in buffer form, but you can drop the buffer syntax since asset ids internally are buffers.

```suggestion
    const assetId = Buffer.from(request.data.assetId, 'hex')
```